### ISSUE TITLE
bug(windows): Keyboard doesn't turn on when switching to FLEx from another Windows program

### ISSUE THREAD

**Issue by rmlockwood** [keymanapp/keyman#6025]
**Describe the bug**
The keyman keyboard that should be turned on when on text box in FLEx for a certain writing system doesn't turn on. 

**To Reproduce**
1. Go to FLEx and the lexicon area
2. Do a find operations (ctl+F) and make sure the cursor is in the find box
3. Switch the Keyman Keyboard to something like Eng (that isn't what should be on for the FLEx writing system)
4. Switch to a different app in Windows
5. Switch back to FLEx
6. The correct keyboard doesn't turn on.

**Expected behavior**
I expected the keyman keyboard that is assigned to the applicable writing system that I'm searching for to turn on.

**Comment by mcdurdin** [comment]
Reported Keyman version: 14.0.277
Reported FLEx version: ??? (9.0?)

I have reproduced this issue with Keyman 15.0.146-alpha and FLEx 9.0.17.455 64 bit.

Debug log: [system0.zip](https://github.com/keymanapp/keyman/files/7674716/system0.zip)

**Comment by mcdurdin** [comment]
I have reviewed the log here and found the following:

1. Keyman is responding correctly to system keyboard change requests
2. I cannot tell where the system keyboard change request comes from -- whether Flex or another component

More importantly, however, I discovered that this issue also arises with Windows keyboards, so it does not appear to be a Keyman issue. We might need to raise this with the FLEx team (@ermshiperete @KenZook any comment?)
